modules/connectors/ninja_van/karrio/providers/ninja_van/tracking.py

```diff
@@ -29,6 +30,7 @@ def parse_tracking_response(
 def _extract_details(
     data: dict,
     settings: provider_utils.Settings,
+        ctx: dict
 ) -> models.TrackingDetails:
     details = None  # parse carrier tracking object type
```

Comment:
You need to arrange a response for tracking 
Like Karrio have own response structure 
You need to get all responses from the ninja van and set each value to karrio response
which you need to show in the karrio response

Reply:
Check this other extensions for ref 
I think FedEx is a great example

Reply:
Done